### DISCUSSION TITLE
One expanded Disclosure at a time

### DISCUSSION THREAD

**Discussion by loonskai** [ariakit/ariakit#4248] [Questions]
Hi, what are you recommendations on implementing a set of Disclosure items that would be in sync in a way that only single item can be expanded at a time? Here is an example of such behavior https://getbootstrap.com/docs/4.1/components/collapse/#accordion-example
I was trying to share a single Disclosure store (by analogy with the [Navigation Menubar](https://ariakit.org/examples/menubar-navigation) example) but couldn't make it work in a proper way. I think I need to track the current active Disclosure ID but maybe there's a better way to achieve such behavior. Thanks!

**Comment by diegohaz** [comment]
I'd use native disclosure elements. See https://developer.mozilla.org/en-US/docs/Web/HTML/Element/details#multiple_named_disclosure_boxes

**Comment by loonskai** [comment]
@diegohaz thanks for your answer. Unfortunately I can't apply it for my case since I need it to be a controlled component. For now I'm going to implement a custom `DisclosureGroup` to handle ids of the nested `Disclosure` items.
